index.bs

```diff
@@ -1770,12 +1770,15 @@ following additional fields:
     display could return this value as 1.6 to indicate its preferred content
     scaling. Default is none.
 
-: color-gamuts (optional)
+: color-gamut (optional)
 :: An optional field indicating what color spaces can be decoded and rendered by
```

Comment:
I would consider rephrasing this to make it clear that it is the widest color space that the client supports. 

```
An optional field indicating the widest color space that can be decoded and rendered by
the media receiver. The media sender may use this value to determine how to encode
video, and may assume that all narrower color spaces are also supported.
```

Reply:
I know you do it below with the `NOTE:` section so feel free to ignore this advice if you prefer that approach instead.

Reply:
Done.